pkg/scheduler/service/scheduler_test.go

```diff
@@ -19,65 +19,60 @@ import (
 	"github.com/kyma-incubator/reconciler/pkg/scheduler/reconciliation/operation"
 )
 
-var (
-	dbConn db.Connection
-	mu     sync.Mutex
-)
-
-func TestScheduler(t *testing.T) {
-	t.Run("Test run once", func(t *testing.T) {
-		// run once will already expect the reconciliation to be in progress as there is no scheduling in place
-		clusterState := testClusterState("testCluster", 1, model.ClusterStatusReconciling)
-		reconRepo := reconciliation.NewInMemoryReconciliationRepository()
-		scheduler := newScheduler(logger.NewLogger(true))
-		require.NoError(t, scheduler.RunOnce(clusterState, reconRepo, &SchedulerConfig{}))
-		requiredReconciliationEntity(t, reconRepo, clusterState)
-	})
+func (s *reconciliationTestSuite) TestSchedulerRunOnce() {
+	t := s.T()
+	// run once will already expect the reconciliation to be in progress as there is no scheduling in place
+	clusterState := testClusterState("testCluster", 1, model.ClusterStatusReconciling)
+	reconRepo := reconciliation.NewInMemoryReconciliationRepository()
+	scheduler := newScheduler(logger.NewLogger(true))
+	require.NoError(t, scheduler.RunOnce(clusterState, reconRepo, &SchedulerConfig{}))
+	requiredReconciliationEntity(t, reconRepo, clusterState)
+}
 
-	t.Run("Test run", func(t *testing.T) {
-		reconRepo := reconciliation.NewInMemoryReconciliationRepository()
-		scheduler := newScheduler(logger.NewLogger(true))
+func (s *reconciliationTestSuite) TestSchedulerRun() {
+	t := s.T()
+	reconRepo := reconciliation.NewInMemoryReconciliationRepository()
+	scheduler := newScheduler(logger.NewLogger(true))
```

Comment:
if you use the testLogger here you dont have to initialize a new one

Reply:
used the test logger in this suite